server.js

```diff
@@ -1,3 +1,8 @@
+// Always load Application Insights before anything else
+const { initApplicationInsights } = require('./common/utils/appinsights')
```

Comment:
the import order here is causing linter errors, but i'm not sure why... any ideas?

Reply:
So our linting rules declare that external packages get required/imported first, then locals. All requires should occur at the top of the files too, I think before any other jazz.
I think you could require and call in one line:
```
({ initApplicationInsights } = require('./common/utils/appinsights'))()
```
although thats from my head; it looks like it might work 😬